packages/utilities/src/BaseComponent.ts

```diff
@@ -153,7 +235,7 @@ function _makeSafe(obj: BaseComponent<any, any>, prototype: Object, methodName:
         if (prototypeMethod) {
           retVal = prototypeMethod.apply(this, arguments);
         }
-        if (classMethod) {
+        if (classMethod !== prototypeMethod) {
```

Comment:
This is an important bug fix; Previously, it would call the prototypeMethod twice if the subclass doesn't implement it.